### PR TITLE
Minor Update of Italian Translation

### PR DESCRIPTION
**Pull request by MishaBoar**
Minor updated to a few strings of the italian translation of 1.14.7.

Builds and works fine in Windows/Ubuntu.

### CODE REVIEW

**Comment by patricklodder** [comment]
Could you try to rebase on top of the current 1.14.7-dev? You're missing the last updates because your commit sits on top of 9bbf8c8363.

I can do it for you if you wish

**Comment by MishaBoar** [comment]
Hey Patrick, my apologies, I had not synced my local 1.14.7-dev repository. 

I tried to rebase, but I am not sure this is the appropriate way to do it for the repo. Let me know if there is anything I should do (I can also restart with a new pull request). Sorry to complicate things further.

**Comment by patricklodder** [comment]
It's fine as it is now, thanks 👍

**Comment by patricklodder** [comment]
Could use a quick review from an Italian speaker, added "help wanted" tag.

File: src/qt/locale/bitcoin_it.ts [hunk 1]
```diff
@@ -2707,7 +2713,7 @@ Use this functionality with extreme caution.</source>
     </message>
     <message>
         <source>If the custom fee is set to 1000 koinu and the transaction is only 250 bytes, then &quot;per kilobyte&quot; only pays 250 koinu in fee, while &quot;total at least&quot; pays 1000 koinu. For transactions bigger than a kilobyte both pay by kilobyte.</source>
-        <translation type="unfinished"></translation>
+        <translation>Se la commissione di transazione è impostata su 1000 koinu e la transazione é solo 250 bytes, la commissione sará solamente di 250 koinu per kilobyte, mentre il totale sará di almeno 1000 koinu. Per transazioni più grandi di un kilobyte entrambi pagano per kilobyte.</translation>
```
Comment (oshawa-connection) [inline src/qt/locale/bitcoin_it.ts:2716]: There are a couple mistakes in this translation; é should be è, sará should be sarà (this appears twice)

Comment (oshawa-connection) [reply src/qt/locale/bitcoin_it.ts:2716]: Otherwise I couldn't find any other mistakes

Comment (MishaBoar) [reply src/qt/locale/bitcoin_it.ts]: Fantastic suggestion, which allowed me to catch also the same issue in some previous phrases (some coming from Bitcoin, I believe). This happens when you are an Italian speaker not using an Italian keyboard and you do not pay enough attention.

**Comment by patricklodder** [comment]
@MishaBoar if you squash these commits I think this is good to go.

**Comment by MishaBoar** [comment]
OK, done! Thank you!

**Review by patricklodder — APPROVED** [review]
ACK - code reviewed

Language validation by @oshawa-connection

**Pull request merged**